### PR TITLE
add queue to ytplayer

### LINKED ISSUE #49: Queue for play command
There should be a queue for requested videos to play by bot and also a set of commands to control it like 
`queue` - to show all queued things
`skip` - to ignore current thing and go to next thing
`queue remove <number>` - to get rid of unwanted elements

### PR DESCRIPTION
**Pull request by szyjah**
resolve #49

### CODE REVIEW

File: ytplayer.py [hunk 1]
```diff
@@ -75,12 +93,61 @@ async def _resume(self,ctx: commands.Context):
...
+            except ValueError:
+                await ctx.send("you idiot itsa not a numbha")
+        
+        
+    
+    @commands.command(name='clear')
+    async def _clear(self,ctx: commands.Context):
+        """Empty queue"""
+        
+        self.qeueu = []
```
Comment (Niewidzialny84) [inline ytplayer.py:134]: You misspelled `queue`

File: ytplayer.py [hunk 2]
```diff
@@ -22,47 +23,64 @@ def cog_check(self, ctx: commands.Context):
...
-            source = discord.FFmpegPCMAudio(url, **FFMPEG_OPTIONS)
-            self.voice.play(source)
-            while(self.voice.is_playing() or self.voice.is_paused()):
-                await asyncio.sleep(1)
-            await self.voice.disconnect()                    
 
+        if not ctx.message.author.voice:
+            await ctx.send("You need to be in a vc to use this command")
+        else:
+            channel = ctx.message.author.voice.channel
```
Comment (Niewidzialny84) [inline ytplayer.py:35]: It contains itself in above if

**Review by Niewidzialny84 — CHANGES_REQUESTED** [review]

**Review by Niewidzialny84 — APPROVED** [review]
Seems legit

**Review by Niewidzialny84 — APPROVED** [review]
Noice, `shuffle` and `remove` is well appreciated in the music queue

File: ytplayer.py [hunk 3]
```diff
@@ -75,12 +91,83 @@ async def _resume(self,ctx: commands.Context):
...
+
+        if len(self.queue) == 0:
+            await ctx.send("Queue is empty you dingus")
+        else:
+            number = 1
+            string = ""
+            for x in self.queue:
+                string = string + str(number) + ". " + x.title + "\n"
+                number = number + 1
+            await ctx.send(string)
```
Comment (seker212) [inline ytplayer.py:144]: 1) Don't we want to keep it in embed like the rest?
2) Suggestion: Maybe we should print only a few first entries for queue and it's length?

**Review by seker212 — CHANGES_REQUESTED** [review]
2 suggestions, generally all right

**Review by seker212 — APPROVED** [review]

**Pull request merged**